modules/config-fields-legacy.adoc

```diff
@@ -16,4 +16,6 @@ Some fields are deprecated or obsolete:
 **Example**: `<1.8.0` +
 **Default:** `<1.6.0`  
 | **DOCUMENTATION_ROOT** | String | Root URL for documentation links
+| **SECURITY_SCANNER_V4_NAMESPACE_WHITELIST** | String | The namespaces to which the security scanner should be enabled for
```

Comment:
I think this might have been deprecated IIRC and security scanning is just on by default